### ISSUE TITLE
The model and loaded state dict do not match exactly

### ISSUE THREAD

**Issue by vobecant** [SwinTransformer/Swin-Transformer-Object-Detection#26]
Hi,

I try to load the Swin backbone using the configuration `mask_rcnn_swin_small_patch4_window7_mstrain_480-800_adamw_3x_coco.py` and weights `swin_small_patch4_window7_224.pth` and I get the following warnings:

```
mmdet - WARNING - The model and loaded state dict do not match exactly

unexpected key in source state_dict: norm.weight, norm.bias, head.weight, head.bias, layers.0.blocks.1.attn_mask, layers.1.blocks.1.attn_mask, layers.2.blocks.1.attn_mask, layers.2.blocks.3.attn_mask, layers.2.blocks.5.attn_mask, layers.2.blocks.7.attn_mask, layers.2.blocks.9.attn_mask, layers.2.blocks.11.attn_mask, layers.2.blocks.13.attn_mask, layers.2.blocks.15.attn_mask, layers.2.blocks.17.attn_mask

missing keys in source state_dict: norm0.weight, norm0.bias, norm1.weight, norm1.bias, norm2.weight, norm2.bias, norm3.weight, norm3.bias
```

I understand the problem with the norm layers. In the original Swin backbone, there is only one normalization layer at the output while there is a norm layer at every output stage in the Swin backbone used for detection.
However, I am not sure why there is the problem with `attn_mask`s. 

Can you please help me?

Thank you very much in advance.

**Comment by hukaixuan19970627** [comment]
I have the same problem with yours. Did you solve it?

**Comment by mstarget** [comment]
I have the same problem as you. Did you solve it?

**Comment by mmaaz60** [comment]
@zeliu98  your comments will be appreciated on this thread, thanks

**Comment by nikhil031294** [comment]
> I have exactly same issue. Do you guys know if it impacts the final accuracy?

Same question! My training loss is not reducing

**Comment by valencebond** [comment]
same question

**Comment by weiyx16** [comment]
This message is ok. The normalizations are re-initialized for FPN and attn_masks are not needed here.
